Update project roadmap (#6)
kubernetes-sigs/testgrid-json-exporter: README.md

# K8s-Testgrid JSON Exporter

> This project is work in progress and not yet in use!

Reviewer: ```suggestion
This project is a work in progress and not ready for production usage.
```

You should be able to access the exposed metrics http://127.0.0.1:8081

## Roadmap

> - [X] Configure custom testgrid json exporter container

Reviewer: we can remove the completed ones, sometime in the future

Author: Yes! Maybe after it's running in prod.
